anchor-reference-server/src/main/java/org/stellar/anchor/reference/event/AnchorEventProcessor.java

```diff
@@ -24,12 +24,31 @@ public class AnchorEventProcessor {
 
   public void handleQuoteEvent(QuoteEvent event) {
     Log.debug(String.format("Received quote event: %s", event));
+    switch (event.getType()) {
+      case "quote_created":
+        break;
+      default:
+        Log.debug("error: anchor_platform_event - invalid message type '%s'%n", event.getType());
+    }
   }
 
   public void handleTransactionEvent(TransactionEvent event) {
     Log.debug(String.format("Received transaction event: %s", event));
+    switch (event.getType()) {
+      case "transaction_created":
+      case "transaction_payment_received":
```

Comment:
There is a modification https://github.com/stellar/stellar-anchor-platform/blob/main/design/eventSchema/event.yml#L54 

We do not have "transaction_payment_received" event type any more.

Reply:
will address the schema change in a separate PR